samples/openapi3/server/petstore/springboot-useoptional/src/main/java/org/openapitools/model/AdditionalPropertiesArray.java

```diff
@@ -25,25 +27,24 @@
 public class AdditionalPropertiesArray extends HashMap<String, List>  {
 
   @JsonProperty("name")
-  private String name;
+  private Optional<String> name = Optional.empty();
 
   public AdditionalPropertiesArray name(String name) {
-    this.name = name;
+    this.name = Optional.ofNullable(name);
     return this;
   }
 
   /**
    * Get name
    * @return name
   */
-  
   @Schema(name = "name", required = false)
-  public String getName() {
+  public Optional<String> getName() {
     return name;
   }
 
-  public void setName(String name) {
-    this.name = name;
+  public void setName(Optional<String> name) {
+    this.name = Objects.requireNonNull(name, "A parameter of type Optional must not be null.");
```

Comment:
Not sure if you really need this check for an Optional. You don't do this for container classes as well